src/getCoinCombination.test.js

```diff
@@ -7,4 +7,24 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return number of pennies`, () => {
```

Comment:
the same as in your previous task - you should specify what should be returned, not only the 'number'